receiver/receiver_input.py

```diff
@@ -0,0 +1,10 @@
+import sys
+import ast
+
+def bms_console_input():
+  data = sys.stdin.readlines()    
+  for i in data:
```

Comment:
The content of this loop doesn't use `i`. It acts on the entire `data`. Is it needed to be in the loop?
Point is to make the naming clear, so that such questions are not needed.